server/handles/ssologin.go

```diff
@@ -132,7 +132,7 @@ func autoRegister(username, userID string, err error) (*model.User, error) {
 		ID:         0,
 		Username:   username,
 		Password:   random.String(16),
-		Permission: int32(setting.GetInt(conf.SSODefaultPermission, 0)),
+		Permission: int32(strconv.ParseInt(setting.GetInt(conf.SSODefaultPermission, 0)), 10, 32),
```

Comment:
There are too many arguments in conversion to int32, and can't pass a int value to `ParseInt`